graylog2-web-interface/src/components/common/Scratchpad.jsx

```diff
@@ -192,7 +186,12 @@ const Scratchpad = () => {
                   spellCheck={false} />
 
         <Footer>
+          <OverlayTrigger placement="right" trigger={['click', 'focus']} overlay={<Tooltip id="scratchpad-help">You can use this space to store personal notes and other information while interacting with Graylog, without leaving your browser window. For example, store timestamps, user IDs, or IP addresses you need in various investigations.</Tooltip>}>
```

Comment:
I guess you can't put both `click` and `focus` into this:
![broken-overlay](https://user-images.githubusercontent.com/448763/88296265-c2316d00-ccfe-11ea-801d-8c6bc13be8ff.gif)

Reply:
Replaced with `[hover,focus]`